crates/yttrium/src/chain_abstraction/client.rs

```diff
@@ -396,13 +411,110 @@ impl Client {
         }
     }
 
-    // pub async fn send() {
-    //     // TODO input signed txns
-    //     // TODO send route first, etc. (possibly in parallel)
-    //     // TODO wait_for_success
-    //     // TODO send initial transaction
-    //     // TODO await initial transaction success
-    // }
+    pub async fn execute(
+        &self,
+        ui_fields: UiFields,
+        route_txn_sigs: Vec<PrimitiveSignature>,
```

Comment:
don't you need a chain id and the whole transaction with a signature to send a raw transaction 🤔

Reply:
The unsigned txn is passed in UiFields, so I build the signed txn inside here